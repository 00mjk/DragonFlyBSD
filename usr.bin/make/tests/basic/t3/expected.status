$DragonFly: src/usr.bin/make/tests/basic/t3/expected.status,v 1.1 2005/02/26 10:43:09 okumoto Exp $
2
