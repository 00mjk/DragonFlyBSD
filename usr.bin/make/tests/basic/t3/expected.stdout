$DragonFly: src/usr.bin/make/tests/basic/t3/expected.stdout,v 1.1 2005/02/26 10:43:09 okumoto Exp $
